js/pages/configuration/configuration.js

```diff
@@ -120,7 +121,8 @@ define([
       const info = await buildInfoService.getBuildInfo();
       await sourceApi.initSourcesConfig();
       super.onPageCreated();
-      this.isSolrVocabEnabled = info.configuration.vocabulary.solrEnabled;
+      this.isSolrVocabEnabled = info.configuration.vocabulary.solrEnabled && info.configuration.vocabulary.hasOwnProperty("cores");
+      this.isSolrVocabCoreAvailable = false;
```

Comment:
At the top of your object, you've defined isSolrVocabEnabled as an observable, but on this line (126) you are overwriting it with a  boolean.  So, if it's not important to have this value observable (ie: have the UI respond to changes in this value) then just declare it to default to 'false'.   Or, if you do want the UI to respond to changes in the value of that variable: then you should say `this.isSolrVocabCoreAvailable(false)`...btw, just looking at your coe changes, it seems that it is always false...unless I'm missing whre this.isSolrVocabCoreAvailable is set to true somewhere?